05-Jason-Luo/wk07/3-wed/image-search-fix/js/search.js

```diff
@@ -0,0 +1,126 @@
+// console.log($.fn.jquery, _.VERSION);
+var pageNumber = 1;
```

Comment:
```suggestion
let pageNumber = 1;
```
Nooo, please not this old villain!